src/pages/storage/actions/DuplicateVolumeBtn.tsx

```diff
@@ -11,7 +11,9 @@ interface Props {
 }
 
 const DuplicateVolumeBtn: FC<Props> = ({ volume, className, onClose }) => {
-  const { openPortal, closePortal, isOpen, Portal } = usePortal();
+  const { openPortal, closePortal, isOpen, Portal } = usePortal({
+    closeOnOutsideClick: false,
+  });
```

Comment:
The custom select should contain the click events and avoid propagation. I think this (and similar) workarounds should be removed before merging. Again, we can wait for a fixing PR for the custom select to land until we come back to this one.

Reply:
I didn't find an easy way to avoid this. Maybe worth keeping the current approach from this PR.

Reply:
I found a way to stop the event in the custom select as part of #1002 
please remove this workaround -- from here and also from other files, after that fix is merged and this is rebased on main.

Reply:
@Kxiru #1002 has been merged